### ISSUE TITLE
beginner documentation

### ISSUE THREAD

**Issue by dbinetti** [thefrontside/ember-impagination#53]
i'm just learning ember, so perhaps trying this library was a bit more than i could handle, but as a newcomer i'm totally lost by the documentation.  a simplified example that matches the dummy app would be very helpful.

**Comment by Robdel12** [comment]
I agree 100%. We've been pretty busy lately with client work. But this needs to happen sooner rather than later.
